marco/AutamationPractice/src/main/java/at05ui/sampleapp/ui/SearchPage.java

```diff
@@ -0,0 +1,40 @@
+package at05ui.sampleapp.ui;
+
+import org.openqa.selenium.WebDriverException;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class SearchPage extends BasedPageObject {
+
+  @FindBy(name = "Submit")
+  WebElement Submit;
+  @FindBy(xpath = "//div[2]/a/span")
+  WebElement btnAdd;
+
+  @FindBy(xpath = "//div[@id='layer_cart']/div/div[2]/div[4]/a/span")
+  WebElement btnProcced;
+
+  public void clickBtnAdd() {
+    try {
+      Thread.sleep(10000);
```

Comment:
You should evit use sleep

Reply:
This method It was of test jejejejeje, but I already removed it.